docs/src/main/asciidoc/kotlin.adoc

```diff
@@ -5,7 +5,6 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Using Kotlin
 
-:extension-status: preview
```

Comment:
Please also remove `include::{includes}/extension-status.adoc[]` a few lines below.

Reply:
Thanks, done.

Reply:
@gsmet I see you asked the same in https://github.com/quarkusio/quarkus/pull/26709. There is a nice message when the extension status is stable, what's the reason behind removing it? 

![image](https://user-images.githubusercontent.com/54133/180044590-742b0684-ee86-4760-ac77-24eeb57cdaa4.png)

Reply:
@gastaldi we always only included the note in non-stable extensions so I want us to stay consistent.